crates/pindakaas/src/bool_linear.rs

```diff
@@ -153,9 +179,9 @@ pub struct LinearEncoder<Enc = StaticLinEncoder, Agg = BoolLinAggregator> {
 /// of using the [`BoolLinAggregator`], and are generally the required input
 /// type for encoders of boolean linear constraints.
 pub struct NormalizedBoolLinear {
-	terms: Vec<Part>,
-	cmp: LimitComp,
-	k: PosCoeff,
+	pub(crate) terms: Vec<Part>,
```

Comment:
made pub(crate) since I put some encoders back into their own file, otherwise bool_linear will be huge

Reply:
TODO for Jip